### PR TITLE
Add Example Project and First Example

### PR DESCRIPTION
**Pull request by dcvz**
This adds a basic project setup with an example of using the queue setup and tracking progress via the `ProgressComponent` 💃 

This sets a place for users to submit more examples and I'll be adding more over time. It'll also be a good resource for reproducing and fixing reported bugs.

### CODE REVIEW

**Comment by dcvz** [comment]
@Guichaguri can you verify this runs correctly on Android? I want to merge this ASAP so I can start working on more examples + bug fixes.

File: example/react/screens/PlaylistScreen.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import TrackStore, { playbackStates } from '../stores/Track';
+
+@observer
+export default class LandingScreen extends Component {
+  static navigationOptions = {
+    title: 'Playlist Example',
+  };
+
+  componentDidMount() {
+    TrackPlayer.setupPlayer();
```
Comment (Guichaguri) [inline example/react/screens/PlaylistScreen.js]: You should retrieve the player state or enable the [stopWithApp](https://github.com/react-native-kit/react-native-track-player/wiki/Documentation#updateoptionsdata) option

File: example/index.js [hunk 2]
```diff
@@ -9,12 +9,13 @@ AppRegistry.registerComponent('example', () => App);
...
-    TrackStore.artist = track.artist;
-    TrackStore.artwork = track.artwork;
+    if (data.nextTrack) {
+      const track = await TrackPlayer.getTrack(data.nextTrack);
+      TrackStore.title = track.title;
+      TrackStore.artist = track.artist;
+      TrackStore.artwork = track.artwork;
+    }
   } else if (data.type === 'playback-state') {
-    PlayerStore.playbackState = data.state;
```
Comment (dcvz) [inline example/index.js]: Was this causing an issue on Android @Guichaguri ? this is what updates the state label in the playlist view.

Comment (Guichaguri) [reply example/index.js]: No, it doesn't make sense to set the playbackState here and then set it again right after it. (Lines 19 and 21)

**Pull request merged**